src/components/SearchList.js

```diff
@@ -1,11 +1,46 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import styled from 'styled-components';
 import propTypes from 'prop-types';
 
-function SearchList({ items, handleSearchQueryFromList, loading }) {
+function SearchList({
+  items,
+  activeSearchListElement,
+  onSetActiveSearchListElement,
+  setSelectedCity,
+  handleSearchQueryFromList,
+  loading,
+}) {
+  const form = document.querySelector('div #form');
+
+  const handleKeyDown = (e) => {
+    e.stopImmediatePropagation();
+
+    if (e.keyCode === 38 && activeSearchListElement > 0) {
```

Comment:
Please create const for numbers representing keys, something like `const enterKey = some number` so that it's easier to understand what numbers mean.

Reply:
Fixed